crates/ty_python_semantic/src/types/enums.rs

```diff
@@ -11,44 +13,52 @@ use crate::{
     },
 };
 
-#[derive(Debug, PartialEq, Eq)]
-pub(crate) struct EnumMetadata {
-    pub(crate) members: FxOrderSet<Name>,
-    pub(crate) aliases: FxHashMap<Name, Name>,
+#[salsa::interned(debug, heap_size=EnumMetadata::heap_size)]
+pub(crate) struct EnumMetadata<'db> {
+    #[returns(ref)]
+    pub(crate) members: FxOrderMap<Name, Type<'db>>,
+    #[returns(ref)]
+    pub(crate) aliases: BTreeMap<Name, Name>,
```

Comment:
changed from `FxHashMap` to `BTreeMap` because `FxHashMap` doesn't implement `HashEqLike`, which it looks like is required for salsa to intern the struct